### ISSUE TITLE
App rejected Due to copy right issue

### ISSUE THREAD

**Issue by Imran4424** [PierfrancescoSoffritti/android-youtube-player#608]
My app is rejected saying that, 

Webviews
We don’t allow apps whose primary purpose is to provide a webview of a website, regardless of ownership, or to aggregate content that does not belong to the developer.

What should I do now?

**Comment by AntonJiang** [comment]
Is your app targeting some specific website or youtube channel?

**Comment by PierfrancescoSoffritti** [comment]
Yeah it shouldn't be an issue if you're just accessing YouTube in general. Are you doing something else? do you have other webviews?

**Comment by Imran4424** [comment]
no, I am just using YouTube
disabling YouTube Button

**Comment by Imran4424** [comment]
[This is my app link](https://play.google.com/store/apps/details?id=com.luminous.android.cartoongallery)

I submit an update with a signatured APK file

because YouTube Player was not playing in some devices

then, that update get rejected

**Comment by Imran4424** [comment]
https://support.google.com/googleplay/android-developer/topic/9876964#impersonation-intellectual-property

they give me this link

**Comment by PierfrancescoSoffritti** [comment]
uhm, looking at that link i would say the problem is with your app, not with the player itself. Share the apk if you want, can have a look.
